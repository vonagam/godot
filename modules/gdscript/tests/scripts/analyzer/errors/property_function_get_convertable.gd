var prop: String:
	get = get_prop

func get_prop() -> StringName:
	return &'prop'

func test():
	print('not ok')
