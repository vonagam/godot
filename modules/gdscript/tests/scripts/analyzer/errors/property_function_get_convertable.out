GDTEST_ANALYZER_ERROR
Cannot use "get_prop" as getter: return type "StringName" is not compatible with property type "String".
